demos/mediapipe/holistic_tracking/README.md

```diff
@@ -67,4 +67,60 @@ python mediapipe_holistic_tracking.py
 ```
 
 ## Output image
-![output](output_image.jpg)
\ No newline at end of file
+![output](output_image.jpg)
```

Comment:
Need description in readme - this is to run on input umages - this part allows for running on input video - this is to run on camera.